nixos/modules/services/hardware/nvidia-container-toolkit/cdi-generate.nix

```diff
@@ -0,0 +1,37 @@
+{ config, pkgs }: let
+  mountOptions = { options = ["ro" "nosuid" "nodev" "bind"]; };
+  mounts = [
+    { hostPath = "${config.hardware.nvidia.package.bin}/bin/nvidia-cuda-mps-control";
+      containerPath = "/usr/bin/nvidia-cuda-mps-control"; }
+    { hostPath = "${config.hardware.nvidia.package.bin}/bin/nvidia-cuda-mps-server";
```

Comment:
`nvidia_x11_beta_open` seems to only provide `outputs = [ "out" ]`; we can use `lib.getBin p` instead of `p.bin`